cds-snc/notification-terraform
# Adding alarm for when karpenter replicas are unavailable

# Summary | Résumé

Adding alarm for when karpenter is not running properly. 

# Test instructions | Instructions pour tester la modification

Tested in dev
- Break karpenter deployment (set node selector in karpenter deployment to something that doesn't exist and apply)
- Verify that there are unavailable pods (pods in pending state)
- Wait for 10 minutes (two eval periods)
- Verify that the karpenter alarm is in alarm in aws console
- (In staging, verify there's an alert in slack staging-ops)
